SingularityBase/src/main/java/com/hubspot/singularity/api/SingularityDeployRequest.java

```diff
@@ -12,15 +13,22 @@
   private final Optional<Boolean> unpauseOnSuccessfulDeploy;
   private final SingularityDeploy deploy;
   private final Optional<String> message;
+  private final Optional<SingularityRequest> newRequestData;
 
   @JsonCreator
   public SingularityDeployRequest(
       @JsonProperty("deploy") SingularityDeploy deploy,
       @JsonProperty("unpauseOnSuccessfulDeploy") Optional<Boolean> unpauseOnSuccessfulDeploy,
-      @JsonProperty("message") Optional<String> message) {
+      @JsonProperty("message") Optional<String> message,
+      @JsonProperty("newRequestData") Optional<SingularityRequest> newRequestData) {
```

Comment:
is there a reason this should be here, as opposed to on the `SingularityDeploy` object?